README.md

```diff
@@ -394,6 +394,55 @@ Set the fields for the CSV export function. By default, ng-admin uses the fields
                 .stripTags(true)
         ]);
 
+
+### batchDeleteView Settings
+
+* `singleApiCall(function(entityIds) {})`
+
+By default, the batch delete action make as many requests as you selected objects. If you want to delete all selected objects in a single request, you have to define a `singleApiCall` function. You can use it if you API support filter for multiple values on a DELETE request.
+
+        // Will call DELETE /posts?_filters=id1,id2,.._
+        post.batchDeleteView()
+            .singleApiCall(function (ids) {
+            return postIds.join(,);
```

Comment:
Sorry, I just noticed that what I ask is possible. Then I don't see why we support and document) another, less usable syntax. Returning an object literal should be the only way.